x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/data/BooleanArrayVector.java

```diff
@@ -80,6 +84,6 @@ public String toString() {
 
     @Override
     public void close() {
-        // no-op
+        super.close();
```

Comment:
These `void foo() { super.foo(); }` calls always confuse me. Maybe just remove it?

Reply:
I'll take a look at this.

Reply:
Removed superfluous super calls